src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs

```diff
@@ -22,6 +22,7 @@ internal class GraphAuthenticationProvider : IAuthenticationProvider
         private const string AuthorizationHeaderKey = "Authorization";
         readonly IAuthorizationHeaderProvider _authorizationHeaderProvider;
         readonly GraphServiceClientOptions _defaultAuthenticationOptions;
+        private readonly string[] _graphUris = ["graph.microsoft.com", "graph.microsoft.us", "dod-graph.microsoft.us", "graph.microsoft.de", "microsoftgraph.chinacloudapi.cn", "canary.graph.microsoft.com", "graph.microsoft-ppe.com"];
```

Comment:
how do we know these are all the URIs?

Reply:
@jennyf19 great question - check out [AzureIdentityAccessTokenProvider.cs](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/5915858fccbb0ff0f7829c27c722bd2dcabc2e48/src/Microsoft.Graph.Core/Authentication/AzureIdentityAccessTokenProvider.cs#L19) from the Graph repo, shared by a friend from Graph @andrueastman